src/models/Expense.model.js

```diff
@@ -1,9 +1,52 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    amount: {
+      type: DataTypes.DECIMAL(10, 2),
+      get() {
+        return parseFloat(this.getDataValue('amount'));
+      },
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      references: {
+        model: 'Users',
+        key: 'id',
+      },
+      allowNull: false,
+    },
+    category: {
+      type: DataTypes.STRING,
```

Comment:
Consider changing the `category` field to a foreign key if it is intended to reference a `Category` model. This will ensure referential integrity and better data management.